src/expr/src/expr/mod.rs

```diff
@@ -205,8 +205,8 @@ impl ExprContext {
         Self { curr_epoch }
     }
 
-    pub fn get_physical_time(&self) -> u64 {
-        self.curr_epoch.physical_time()
+    pub fn get_unix_millis(&self) -> u64 {
+        self.curr_epoch.as_unix_millis()
```

Comment:
What about including `* 1000` and output micros directly here?

Reply:
ok.